multiplex.py

```diff
@@ -5,8 +5,15 @@
 import copy
 import traceback
 import os
+import logging
 
 from jsonschema import validate
+from jsonschema import exceptions
+
+EC_SUCCESS=0
+EC_SCHEMA_FAIL=1
+EC_JSON_FAIL=2
+EC_REQUIREMENTS_FAIL=3
```

Comment:
so I'm ignorant...what does EC mean?

Reply:
> so I'm ignorant...what does EC mean?

Exit Code --> enumerate exit codes for consistency